src/SFML.Window/Cursor.cs

```diff
@@ -122,9 +122,14 @@ public enum CursorType
         /// the operating system.
         /// </summary>
         /// <param name="type">System cursor type</param>
+        /// <exception cref="LoadingFailedException" />
         public Cursor(CursorType type)
             : base(sfCursor_createFromSystem(type))
         {
+            if (CPointer == IntPtr.Zero)
+            {
+                throw new LoadingFailedException("cursor");
```

Comment:
The error message generated from this constructor could be bit misleading, but I think it's okay